docs/sphinx_server.py

```diff
@@ -15,14 +15,13 @@
 from livereload import Server, shell
 
 rebuild_cmd = shell('make html', cwd='.')
+rebuild_root = "_build/html"
+watch_source_dir = "../smqtk_relevancy"
```

Comment:
This is not being used below. L38 is still hard-coded to refer to an old directory from the mono-repo.